server/utils/applicationUtils.ts

```diff
@@ -85,3 +87,99 @@ const nameAnchorElement = (
 const textValue = (value: string) => {
   return { text: value }
 }
+
+export const getStatusTag = (statusLabel: string, statusId: string) => {
+  return `<strong class="govuk-tag govuk-tag--${getStatusTagColour(statusId)}">${statusLabel || 'Received'}</strong>`
+}
+
+const getStatusTagColour = (statusId: string) => {
+  switch (statusId) {
+    case 'f5cd423b-08eb-4efb-96ff-5cc6bb073905':
+      return 'red'
+    case 'ba4d8432-250b-4ab9-81ec-7eb4b16e5dd1':
+      return 'pink'
+    case 'a919097d-b324-471c-9834-756f255e87ea':
+      return 'yellow'
+    case '176bbda0-0766-4d77-8d56-18ed8f9a4ef2':
+      return 'green'
+    case 'fe254d88-ce1d-4cd8-8bd6-88de88f39019':
+      return 'purple'
+    case '9a381bc6-22d3-41d6-804d-4e49f428c1de':
+      return 'grey'
+    case '004e2419-9614-4c1e-a207-a8418009f23d':
+      return 'grey'
+    case 'f13bbdd6-44f1-4362-b9d3-e6f1298b1bf9':
+      return 'grey'
+    case '89458555-3219-44a2-9584-c4f715d6b565':
+      return 'grey'
+    default:
+      return 'blue'
+  }
+}
+
+// [
+//   {
+//     id: 'f5cd423b-08eb-4efb-96ff-5cc6bb073905',
+//     name: 'moreInfoRequested',
+//     label: 'More information requested',
+//     description:
+//       'The prison offender manager (POM) must provide information requested for the application to progress.',
+//     statusDetails: [[Object], [Object], [Object], [Object], [Object], [Object], [Object], [Object], [Object]],
+//   },
+//   {
+//     id: 'ba4d8432-250b-4ab9-81ec-7eb4b16e5dd1',
+//     name: 'awaitingDecision',
+//     label: 'Awaiting decision',
+//     description: 'The CAS-2 team has the information they need and will make a decision.',
+//     statusDetails: [],
+//   },
+//   {
+//     id: 'a919097d-b324-471c-9834-756f255e87ea',
+//     name: 'onWaitingList',
```

Comment:
but of copy and paste leftover here, which was really useful for checking the id's though!
would it be worth putting in a comment a reference to the file we get these from on the API?

Reply:
good shout, have done